g3/lbp_g3.c

```diff
@@ -93,10 +93,49 @@ enum LBP_G3_EAP_STATE {
 };
 
 static struct {
+#if LBP_G3_PAN_COORDINATOR
```

Comment:
Yes, wether we are a PAN_COORDINATOR is determined by a `device_type` field in G3 netif's context. Those ifdefs are used for memory usage optimization, when we know from the start, that we will never be a PAN coordinator. That's, because a PAN coordinator is using much more memory for its arrays.